src/main/java/com/uber/cadence/internal/sync/ExternalWorkflowStubImpl.java

```diff
@@ -59,6 +59,24 @@ public void signal(String signalName, Object... args) {
     }
   }
 
+  @Override
+  public void signalCrossDomain(String signalName, String domain, Object... args) {
```

Comment:
Probably similar change is needed in ChildWorkflowStub?

Reply:
@Groxx maybe you know?

Reply:
I believe that's the case